specification/app/data-plane/Microsoft.App.DynamicSessions/stable/2024-09-01/DynamicSessions.json

```diff
@@ -0,0 +1,884 @@
+          "description": "The execution type of the code execution request."
+        },
+        "networkStatus": {
+          "$ref": "#/definitions/NetworkStatus",
+          "description": "The network status of the code execution session."
+        },
+        "multiArrayStatus": {
+          "$ref": "#/definitions/MultiArrayStatus",
+          "description": "The multi array status of the code execution session."
+        },
+        "code": {
+          "type": "string",
+          "description": "The string of the code to execute, based on CodeInputType."
+        },
+        "timeoutInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Code execution timeout in seconds."
+        }
+      },
+      "required": [
+        "codeInputType",
+        "executionType",
+        "networkStatus",
+        "multiArrayStatus",
+        "code",
+        "timeoutInSeconds"
+      ]
+    },
+    "SessionCodeExecutionResponse": {
+      "type": "object",
+      "description": "The response containing the result of the code execution.",
+      "properties": {
+        "codeInputType": {
+          "$ref": "#/definitions/CodeInputType",
+          "description": "The code input type of the code execution request."
+        },
+        "executionType": {
+          "$ref": "#/definitions/ExecutionType",
+          "description": "The execution type of the code execution request."
+        },
+        "networkStatus": {
+          "$ref": "#/definitions/NetworkStatus",
+          "description": "The network status of the code execution session."
+        },
+        "multiArrayStatus": {
+          "$ref": "#/definitions/MultiArrayStatus",
+          "description": "The multi array status of the code execution session."
+        },
+        "code": {
+          "type": "string",
+          "description": "The string of the code to execute, based on CodeInputType."
+        },
+        "timeoutInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Code execution timeout in seconds."
+        },
+        "id": {
+          "type": "string",
+          "description": "Session code execution id.",
+          "readOnly": true
+        },
+        "status": {
+          "$ref": "#/definitions/CodeExecutionStatus",
+          "description": "The status of the code execution, indicates whether succeeded or not."
+        },
+        "stdout": {
+          "type": "string",
+          "description": "The standard output of the code execution."
+        },
+        "stderr": {
+          "type": "string",
+          "description": "The standard error of the code execution."
+        },
+        "result": {
+          "description": "The result of the code execution."
+        },
+        "executionTimeInMilliseconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The execution time of the code in milliseconds."
+        }
```

Comment:
What is the value of this field if the code execution has not finished? Is it the execution time so far? This should be explained in the description.